src/utils/devLogger.ts

```diff
@@ -3,7 +3,9 @@ export type DevLoggerPrefix =
   | "PUCK_CONFIG"
   | "SAVE_STATE"
   | "THEME_SAVE_STATE"
+  | "LOCAL_VISUAL_CONFIGURATION_DATA"
```

Comment:
We don't use this key when we use the local data in layout mode. We instead use VISUAL_CONFIGURATION_DATA with another log saying it's using localStorage. We should be consistent.